scripts/vaultcli

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+
+NAME=${0##*/}
+
+print_help() {
+  cat << EOM
+$NAME - vault command line
+
+A simple tool for interacting with vault, mainly as a safeguard to make it less
+likely to accidentally overwrite credentials.
+
+USAGE:
+  $NAME <COMMAND> [OPTIONS]...
+
+FLAGS:
+  -h, --help    Prints this help
+
+COMMANDS:
+  read          Reads given vault path in JSON format
+  write         Prompts user with several confirmations before writing to vault
+
+EOM
+}
+
+print_usage() {
+  echo "Usage: $NAME <COMMAND> [OPTIONS]..."
+  echo "Try $NAME [-h, --help] for more info"
+}
+
+print_read_help() {
+  cat << EOM
+USAGE:
+  $NAME read <PATH> [OPTIONS]...
+
+Reads vault credentails in JSON format and prints to standard output.
+
+FLAGS:
+  -h, --help    Print this additional help
+  -o, --output  Save to given output file instead of printing to stdout
+  -r, --raw     Pass output to jq to get a raw value
+
+EOM
+}
+
+print_read_usage() {
+  echo "Usage: $NAME read <PATH> [OPTIONS]..."
+}
+
+print_write_help() {
+  cat << EOM
+USAGE:
+  $NAME write <PATH> <FILE> [OPTIONS]...
+
+Writes given file as data to vault path after passing confirmation. One
+confirmation step is opening the given data file in the 'less' pager so user
+can visually verify the data to be written. Before writing, a backup of
+existing data in vault will be written to a file in the current directory.
+
+FLAGS:
+  -h, --help    Print this additional help
+      --value   Write whole contents of data file into the 'value' key
```

Comment:
I realized that we have some paths in vault that stores entire file content as a blob under the `value` key in that path. We can read using existing `vaultcli read .. -r` functionality, but writes have a different syntax (see L200), so I added a flag to help with writing values.